plugins/Flanger/Noise.cpp

```diff
@@ -31,15 +31,15 @@ namespace lmms
 
 Noise::Noise()
 {
-	inv_randmax = 1.0/FAST_RAND_MAX; /* for range of 0 - 1.0 */
+	inv_randmax = FAST_RAND_RATIO; /* for range of 0 - 1.0 */
```

Comment:
If this variable isn't used anymore, it could be removed

Reply:
I thought of removing that class and integrating it directly in flanger too. I believe that's better for a follow up.

Reply:
Ideally, it shouldn't even be a class.